### DISCUSSION TITLE
[480]0xb40a1abaA585AB6191A295760D08bB65a2aDe9bC

### DISCUSSION THREAD

**Discussion by giscus** [pvpbet/comments#19] [Announcements]
# [480]0xb40a1abaA585AB6191A295760D08bB65a2aDe9bC

🤜 🎲 🤛 イーサリアム仮想マシン(EVM)上に構築された分散型PVPベッティングプラットフォーム。短期的な予測市場に焦点を当てながら、プレイヤーが自分で予測を作成して利益を得ることも可能にします。プレイヤーは、移籍するだけでベッティングに参加できます。結果が確定すると、自動的に報酬が分配されます。プロセス全体を通して、プレイヤーはウォレットを接続したり登録したりする必要がないため、より安全で、シンプルで、より迅速な体験が保証されます。

https://pvpbet.xyz/bets/480/0xb40a1abaA585AB6191A295760D08bB65a2aDe9bC

**Comment by tomumaru** [comment]
This match has been cancelled.
https://news.yahoo.co.jp/articles/6fc25ef8f3e184004d8c738bf968fa2cd2f718ec

**Comment by Siwelfarah** [comment]
Withdraw
